### ISSUE TITLE
Market on Open 

### ISSUE THREAD

**Issue by AndreasClenow** [stefan-jansen/zipline-reloaded#132]
Just a friendly suggestion: 

For some EOD models, it makes sense to place orders as 'market on open', rather than the default Zipline logic of 'market on close'. I quite simple really and I've modded my own setup for this, and I thought might be of interest to the community.

The 'market on close' logic is currently hard coded in the slippage.py file, and I suppose the reason for that is that Quantopian assumed intraday, minute data. 

The simple solution would be to add a field input argument to the constructors in the slippage.py file. Quick, easy and gets the job done.

**Comment by stefan-jansen** [comment]
Good idea @AndreasClenow; will take it on the list for this year's attempts at improving the package.
